roles/config-docker/tasks/main.yml

```diff
@@ -5,3 +5,8 @@
   when: 
   - docker_install|default(False)
 
+- import_tasks: mtu.yml
+  when: 
+  - docker_install|default(False)
+  - check_mtu|default(True)
```

Comment:
Don't think we need the `check_mtu` flag here. It shouldn't hurt to always do this. Also, just use a `block` here and check `docker_install` once